metal3-dev/run.sh

```diff
@@ -1,17 +1,64 @@
 #!/bin/bash -xe
 
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+LOGDIR=${SCRIPTDIR}/logs
+source $SCRIPTDIR/logging.sh
+
+source $SCRIPTDIR/common.sh
+source $SCRIPTDIR/utils.sh
+
 if ! which operator-sdk 2>&1 >/dev/null ; then
-    echo "Did not find operator-sdk, set INSTALL_OPERATOR_SDK=1 in config_$USER.sh"
+    echo "Did not find operator-sdk." 1>&2
+    echo "Install it following the instructions from the baremetal-operator repository." 1>&2
+    exit 1
+fi
+
+if ! which yq 2>&1 >/dev/null ; then
```

Comment:
Note that https://github.com/openshift-metal3/dev-scripts/issues/888 is still open, so I'm not sure anything will install yq on a freshly installed box?

I'd hoped we might remove the dependency by reworking the template mangling in `utils.sh` but if we're adding more usage then we should probably fix that - I guess we could just pip install it, or do pip install --user?

Reply:
I can submit another PR to install it as part of the requirements script, if you think that's reasonable.

Reply:
See #1044